src/appmixer/utils/controls/Each/package.json

```diff
@@ -0,0 +1,7 @@
+{
+	"name": "appmixer.utils.controls.Each",
```

Comment:
This is not used in the fix.

Reply:
Right, that was only needed when doing `appmixer publish` from `Each/` folder. Should it be removed, though? The dependencies for components in `utils:controls` differ a lot.

Reply:
Hmm, I don't see `uuid` in the package.json in appmixer/utils and I don't see any package.json in the appmixer/utils/controls 
So how could this work before? So keep it.
Fix the indentation, should be 2 or 4 spaces, looks you have tabs there.